Description: propagate boolean json type
Apparently the libjson project decided to rename their boolean type to
json_bool...
Author: Nico Golde <nion@debian.org>
Bug-Debian: http://bugs.debian.org/689225

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: vendor
Bug: http://bugs.debian.org/689225
Bug-Debian: http://bugs.debian.org/689225
Forwarded: 

Index: newsbeuter-2.5/src/ttrss_api.cpp
===================================================================
--- newsbeuter-2.5.orig/src/ttrss_api.cpp	2012-09-30 17:25:23.000000000 +0200
+++ newsbeuter-2.5/src/ttrss_api.cpp	2012-09-30 17:43:24.000000000 +0200
@@ -220,7 +220,7 @@
 		const char * link = json_object_get_string(json_object_object_get(item_obj, "link"));
 		const char * content = json_object_get_string(json_object_object_get(item_obj, "content"));
 		time_t updated = (time_t)json_object_get_int(json_object_object_get(item_obj, "updated"));
-		boolean unread = json_object_get_boolean(json_object_object_get(item_obj, "unread"));
+		json_bool unread = json_object_get_boolean(json_object_object_get(item_obj, "unread"));
 
 		rsspp::item item;
 
